kafka-connector/README.md

```diff
@@ -1,3 +1,9 @@
+# Deprecation Notice
+
+***2022-09-26: We have moved this directory to its own repo at
+https://github.com/googleapis/java-pubsub-group-kafka-connector/
+for future releases of the Pub/Sub Kafka connector.*** 
```

Comment:
Can we add: "See https://github.com/GoogleCloudPlatform/pubsub/discussions/325 for the announcement."